### PR TITLE
slimerjs 0.10.0

### PR DESCRIPTION
**Pull request by jefleponot**
Hi,I never use travis CI so ... be patient #1573

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Ahaha it's ok  @jefleponot, we are all here to learn :) 

Let's see what happens with your build now :cat2:

**Comment by jefleponot** [comment]
Hi @mickaelandrieu,

I can't manage adding slimerJS 0.9.6 but slimerJS 0.10.0 works perfectly with @laurentj advices.

If you have any idea about adding slimerJS 0.9.6 ... (trouble with coffee script)

@istr , @BIGjuevos : Agree to merge ?

**Comment by BIGjuevos** [comment]
I'm alright with just adding 0.10 for now.  Can we leave the commented 0.9.6 in there as a TODO?

**Comment by jefleponot** [comment]
yes of course

File: modules/casper.js [hunk 1]
```diff
@@ -2702,7 +2703,9 @@ function createPage(casper) {
     };
     page.onResourceRequested = function onResourceRequested(requestData, request) {
         casper.emit('resource.requested', requestData, request);
-    	var checkUrl = utils.ltVersion(phantom.version, '2.1.0') ? utils.decodeUrl(requestData.url) : requestData.url;
+    //	var checkUrl = utils.ltVersion(phantom.version, '2.1.0') ? utils.decodeUrl(requestData.url) : requestData.url;
```
Comment (mickaelandrieu) [inline modules/casper.js]: please remove commented code :)

**Comment by BIGjuevos** [comment]
I'm OK with this.  :+1: Want to stick to 4 eyes rule.  @istr and @mickaelandrieu

**Comment by mickaelandrieu** [comment]
Thank you @jefleponot for contribution and  @BIGjuevos for review :+1:

**Comment by istr** [comment]
> If you have any idea about adding slimerJS 0.9.6 ... (trouble with coffee script)

Yes, that is a known issue. Thanks @laurentj for SlimerJS 0.10 and the [advertising](https://slimerjs.org/) and @jefleponot for the test suite support. :heart: 

![advertising] (http://i.imgur.com/NOKAMdd.png)

**Pull request merged**